modules/rosa-accessing-your-cluster.adoc

```diff
@@ -121,18 +122,21 @@ Created:     May 27, 2020
 +
 [source,terminal]
 ----
-$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443
+$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443 #<1>
 ----
+<1> For {hcp-title} cluster, the port number should be `443`.
```

Comment:
Should this read as "For {hcp-title} clusters" or "For a {hcp-title-first} cluster"?

Might also consider rephrasing for clarity. Something like: "For {hcp-title} clusters, use port 443."

Reply:
Also, please see https://github.com/openshift/openshift-docs/pull/72868#discussion_r1524263756.